content/articles/drop-catching.markdown

```diff
@@ -1,10 +1,26 @@
 ---
-title: Can I use DNSimple for Drop-catching?
-excerpt: Information about DNSimple drop-catching policy.
+title: Can I use DNSimple for Drop catching?
+excerpt: Information about DNSimple drop catching policy.
 categories:
 - DNSimple
+- Domain Registration
+- API
```

Comment:
It looks quite nice.

![screen shot 2015-10-08 at 10 56 13 am](https://cloud.githubusercontent.com/assets/301553/10370040/38a27e1a-6dab-11e5-9473-ff3edc2ba7af.png)

I like the idea of making things available in whatever categories make sense as it can lead to better discoverability, I think. Even I've had trouble tracking down something that I was looking for as I thought it could fit in multiple categories and it took some time to find the right one. Often I just resort to the search bar when I have trouble figuring out what category to look in.